datasets/inspecties/dataset.json

```diff
@@ -0,0 +1,280 @@
+  "language": "nl",
+  "dateCreated": "",
+  "license": "Creative Commons, Naamsvermelding",
+  "hasBeginning": "2017-01-29",
+  "hasEnd": "",
+  "accrualPeriodicity": "maandelijks",
+  "publisher": "OIS",
+  "description": "Binnen de Gemeente Amsterdam wordt op verschillende niveaus inspecties uitgevoerd door verschillende directies en partijen. Bijvoorbeeld inspecties voor het dagelijkse onderhoud of technische inspecties voor vervangingsplannen.  Soms met een gelijke- methodiek en andere momenten een andere methodiek (CROW, NEN, etc.). Onder de dataset “inspecties” vind je diverse inspectie tabellen",
+  "status": "niet_beschikbaar",
+  "keywords": [
+    "inspecties",
+    "crow",
+    "beeldmeetlat",
+    "openbare ruimte",
+    "Stadswerken"
+  ],
+  "crs": "EPSG:28992",
+  "objective": "Bevat monitoring en inspectie data over de kwaliteit van de openbare ruimte van willekeurige meetlocaties binnen de gemeente Amsterdam (o.a. Weesp). ",
+  "temporalUnit": "seconden",
+  "contactPoint": {
+    "name": "",
+    "email": "datapunt@amsterdam.nl"
+  },
+  "tables": [
+    {
+      "id": "monitorbeeldkwaliteit",
+      "type": "table",
+      "title": "Monitorbeeldkwaliteit",
+      "description": "Deze tabel bevat de inspecties die zijn uitgevoerd voor het dagelijkse onderhoud Beheer Openbare Ruimte. Hiervoor wordt merendeels de CROW 380 methodiek gebruikt. Enkele inspecties worden met een ander doel uitgevoerd en hanteren daarom niet de CROW 380 methodiek.",
+      "provenance": "apt_inspecties",
+      "schema": {
+        "$schema": "http://json-schema.org/draft-07/schema#",
+        "type": "object",
+        "additionalProperties": false,
+        "mainGeometry": "geometrie",
+        "identifier": "id",
+        "required": [
+          "id",
+          "schema"
+        ],
+        "display": "id",
+        "properties": {
+          "schema": {
+            "$ref": "https://schemas.data.amsterdam.nl/schema@v1.1.1#/definitions/schema"
+          },
+          "id": {
+            "type": "string",
+            "description": "Unieke aanduiding van het record."
+          },
+          "environment": {
+            "type": "string",
+            "description": "Lijst bevat de volgende eigenschappen. environment Omgevingsparameter waarin de inspectie heeft plaatsgevonden (bijv. amsterdam, amsterdam2019, amsterdamcentrum, etc.)"
+          },
+          "inspectionRoundType": {
+            "type": "string",
+            "description": " "
+          },
+          "inspectionRoundId": {
+            "type": "string",
+            "description": "Het id van de schouwronde"
+          },
+          "inspectionRoundUri": {
+            "type": "string",
+            "description": "Link naar de gehele schouwronde"
+          },
+          "inspectionRoundName": {
+            "type": "string",
+            "description": "Naam van de schouwronde"
+          },
+          "inspectieRoundPeriodeStart": {
+            "type": "string",
```

Comment:
Worden de tijdstippen idd uitgedrukt in seconden? Dan zou het type beter `number` kunnen worden. Als het datetime waarden zijn, dan kan er nog een `"format": "date-time"` bij worden gezet.